yt/chyt/controller/internal/strawberry/controller.go

```diff
@@ -46,6 +46,13 @@ type Controller interface {
 	// Given speclet should have suitable type for the specific controller.
 	// Otherwise, GetOpBriefAttributes may panic.
 	GetOpBriefAttributes(parsedSpeclet any) map[string]any
+
+	// GetScalerTarget checks whether YT operation should be scaled and returns required scaling parameters.
+	// Return `nil` is scaling is not required.
+	GetScalerTarget(ctx context.Context, oplet *Oplet) (*ScalerTarget, error)
```

Comment:
Since we access this method from another goroutine (not from background goroutine), we need to state it in the comment that `this method may be called concurrently ...`